filebeat/docs/redis-input.asciidoc

```diff
@@ -0,0 +1,93 @@
+:type: redis
+
+[[redis-input]]
+=== Redis input
+
+++++
+<titleabbrev>Redis</titleabbrev>
+++++
+
+experimental[]
+
+Use the `redis` input to read entries from Redis slowlogs.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: redis
+  hosts: ["localhost:6379"]
+  username: "$\{redis_user\}"
+  password: "$\{redis_pwd\}"
+----
+
+
+// REVIEWERS: Which options should I show in the example config? Incidentally,
+// in the reference.yml, password shows up twice in the same namespace. This
+// needs to be fixed in the reference.yml file
```

Comment:
we have a ticket for the option being shown twice?

Reply:
Created a ticket here: https://github.com/elastic/beats/issues/6573